_topics/en/soft-fork-activation.md

```diff
@@ -109,6 +109,9 @@ optech_mentions:
   - title: "PR Review Club discussion about burying past soft fork deployments"
     url: /en/newsletters/2021/04/14/#bitcoin-core-pr-review-club
 
+  - title: "Bitcoin Core #21377 and #21686 add taproot activation mechanism and params"
```

Comment:
Maybe s/params/parameters/ (here and in taproot.md)

Reply:
The extra four letters exceeds our character limit:

```
! find _topics/ -type f | while read file ; do bundle exec _contrib/schema-validator.rb _data/schemas/topics.yaml $file || echo Error: $file ; done | grep .
_topics/en/soft-fork-activation.md
The property '#/optech_mentions/25/title' was not of a maximum string length of 75 in schema https://bitcoinops.org/schema/topics
Error: _topics/en/soft-fork-activation.md
Makefile:24: recipe for target 'test-before-build' failed
```

Otherwise, I myself would much prefer to use the whole word.  If *params* is particularly bothersome, let me know and I'll rephrase.

Reply:
Not too bothersome! Just a mild preference for the full word in the link summary. Fine to leave it as is.